src/normalizations/normalize-message.ts

```diff
@@ -275,10 +290,53 @@ const getTagIds = (t: string | undefined, tn: string | undefined): Array<string
 	return [];
 };
 
+const haveReadReceipt = (
```

Comment:
I would add some tests to cover all the possible cases.
Some examples are:
- if there is no participant it should return false
- if there is no folder and there is a linkFolder but permission are 'r' it should return false
ecc...